enterprise/fulltext-addon/src/test/java/org/neo4j/kernel/api/impl/fulltext/integrations/bloom/BloomBackupIT.java

```diff
@@ -0,0 +1,154 @@
+package org.neo4j.kernel.api.impl.fulltext.integrations.bloom;
+
+import org.junit.BeforeClass;
+import org.junit.Rule;
+import org.junit.Test;
+
+import java.io.File;
+
+import org.neo4j.backup.OnlineBackup;
+import org.neo4j.graphdb.GraphDatabaseService;
+import org.neo4j.graphdb.Node;
+import org.neo4j.graphdb.Relationship;
+import org.neo4j.graphdb.RelationshipType;
+import org.neo4j.graphdb.Result;
+import org.neo4j.graphdb.Transaction;
+import org.neo4j.graphdb.factory.GraphDatabaseFactory;
+import org.neo4j.internal.kernel.api.exceptions.KernelException;
+import org.neo4j.kernel.impl.enterprise.configuration.OnlineBackupSettings;
+import org.neo4j.kernel.impl.proc.Procedures;
+import org.neo4j.kernel.internal.GraphDatabaseAPI;
+import org.neo4j.test.rule.DatabaseRule;
+import org.neo4j.test.rule.EmbeddedDatabaseRule;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+import static org.neo4j.kernel.api.impl.fulltext.integrations.bloom.BloomFulltextConfig.bloom_enabled;
+import static org.neo4j.kernel.api.impl.fulltext.integrations.bloom.BloomIT.ENTITYID;
+import static org.neo4j.kernel.api.impl.fulltext.integrations.bloom.BloomIT.NODES;
+import static org.neo4j.kernel.api.impl.fulltext.integrations.bloom.BloomIT.RELS;
+import static org.neo4j.kernel.api.impl.fulltext.integrations.bloom.BloomIT.SET_NODE_KEYS;
+import static org.neo4j.kernel.api.impl.fulltext.integrations.bloom.BloomIT.SET_REL_KEYS;
+import static org.neo4j.ports.allocation.PortAuthority.allocatePort;
+
+public class BloomBackupIT
+{
+    private static int backupPort;
+
+    @BeforeClass
+    public static void portSetup()
+    {
+        backupPort = allocatePort();
```

Comment:
Give each individual test case their own port. This means you'll have to make your database rule `startLazily` as well.